base/libgit2/callbacks.jl

```diff
@@ -28,18 +28,6 @@ function authenticate_ssh(creds::SSHCredentials, libgit2credptr::Ptr{Ptr{Void}},
         username_ptr, schema, host)
     isusedcreds = checkused!(creds)
 
-    errcls, errmsg = Error.last_error()
-    if errcls != Error.None
-        # Check if we used ssh-agent
-        if creds.usesshagent == "U"
-            println("ERROR: $errmsg ssh-agent")
```

Comment:
Technically a separate issue, but is really worth backing this out of this pull request and making a separate commit?